app/controllers/journeys/expenses/WorkFromBusinessPremisesController.scala

```diff
@@ -54,18 +54,20 @@ class WorkFromBusinessPremisesController @Inject() (override val messagesApi: Me
       case Some(value) => formProvider(userType(request.user.isAgent)).fill(value)
     }
     val radioContentString = buildLegendHeadingWithHintString(
-      WorkFromBusinessPremisesPage,
-      Some(userType(request.user.isAgent)),
-      headingExtraClasses = "govuk-!-margin-bottom-2")
+      s"workFromBusinessPremises.title.${userType(request.user.isAgent)}",
+      s"workFromBusinessPremises.hint.${userType(request.user.isAgent)}",
+      headingClasses = "govuk-fieldset__legend govuk-fieldset__legend--l"
+    )
 
     Ok(view(preparedForm, mode, userType(request.user.isAgent), taxYear, businessId, radioContentString))
   }
 
   def onSubmit(mode: Mode): Action[AnyContent] = (identify andThen getData) async { implicit request =>
     val radioContentString = buildLegendHeadingWithHintString(
```

Comment:
All comments with regards to `legendContentString` in `FinancialExpensesController` also apply here.

Reply:
@Jake-Raffe Could we also do the same with pulling the content method out into a shared method here like we did in financial expenses controller?

Reply:
Thanks, missed this one